distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -2870,6 +2870,81 @@ public void testInjectAttributesOnUpdate() throws Exception {
         }
     }
 
+    @Test
+    public void testUploaderPolicyAllowsUploaderToQueryAndUpdateMetacard() throws Exception {
```

Comment:
❗️ This test really validates two things, so we may either want to split it up into different test methods, or extract `private` methods that group and clearly tell what we're testing.